src/SimpleDetailsList.tsx

```diff
@@ -87,16 +89,16 @@ export class SimpleDetailsList extends React.Component<any, IDetailsListBasicExa
     return (
       <div className='ms-DetailsListBasicExample'>
         <Toggle
-          isToggled={ isGridVisible }
+          checked={ isGridVisible }
           onChanged={ isToggled => this.setState({ isGridVisible: isToggled }) }
           label='Grid visible'
           onText='On'
-          offText='Off' 
+          offText='Off'
           />
 
         <CommandBar items={ this._getCommandItems() } />
 
-        <div style={ isGridVisible ? null : { display: "none" } }>
+        { isGridVisible && (

```

Comment:
This is probably the best option here; in your previous implementation, we have no notification that the width changes when you remove display none, so we don't remeasure.